spec/controllers/projects_controller_spec.rb

```diff
@@ -45,42 +45,30 @@
-          "      <meta>\n" \
-          "        <tigerdata:project>\n" \
-          "          <ProjectDirectory>#{project.project_directory}</ProjectDirectory>\n" \
-          "          <Title>#{project.metadata[:title]}</Title>\n" \
-          "          <Description>#{project.metadata[:description]}</Description>\n" \
-          "          <DataSponsor>#{project.metadata[:data_sponsor]}</DataSponsor>\n" \
-          "          <DataManager>#{project.metadata[:data_manager]}</DataManager>\n" \
-          "          <Department>77777</Department>\n" \
-          "          <Department>88888</Department>\n" \
-          "          <DataUser>n/a</DataUser>\n" \
-          "          <ProjectID>abc-123</ProjectID>\n" \
-          "        </tigerdata:project>\n" \
-          "      </meta>\n" \
-          "      <quota>\n" \
-          "        <allocation>500 GB</allocation>\n" \
-          "        <description>Project Quota</description>\n" \
-          "      </quota>\n" \
-          "      <collection cascade-contained-asset-index=\"true\" contained-asset-index=\"true\" unique-name-index=\"true\">true</collection>\n" \
-          "      <type>application/arc-asset-collection</type>\n" \
-          "      <pid>path=/td-test-001/test/tigerdata</pid>\n" \
-          "    </args>\n" \
-          "  </service>\n" \
-          "</request>\n"
-        )
+          xml_doc = Nokogiri::XML(response.body)
+          expect(xml_doc.xpath("/resource/title").text).to eq project.title
+          expect(xml_doc.xpath("/resource/projectDirectory/projectDirectoryPath").text).to eq project.project_directory
+          expect(xml_doc.xpath("/resource/description").text).to eq project.metadata[:description]
+          expect(xml_doc.xpath("/resource/dataSponsor/@userID").text).to eq project.metadata[:data_sponsor]
+          expect(xml_doc.xpath("/resource/dataManager/@userID").text).to eq project.metadata[:data_manager]
+          expect(xml_doc.xpath("/resource/projectID").text).to eq project.metadata[:project_id]
+        end
+
+        it "renders the Mediaflux metadata as xml", :integration do
+          project.approve!(current_user: sponsor_and_data_manager)
+          get :show_mediaflux, params: { id: project.id, format: :xml }
+          expect(response.content_type).to eq("application/xml; charset=utf-8")
+          xml_doc = Nokogiri::XML(response.body)
+          expect(xml_doc.xpath("/meta//Title").text).to eq project.title
+          expect(xml_doc.xpath("/meta//ProjectDirectory").text).to eq project.project_directory
+          expect(xml_doc.xpath("/meta//Description").text).to eq project.metadata[:description]
+          expect(xml_doc.xpath("/meta//DataSponsor").text).to eq project.metadata[:data_sponsor]
+          expect(xml_doc.xpath("/meta//DataManager").text).to eq project.metadata[:data_manager]
+          expect(xml_doc.xpath("/meta//ProjectID").text).to eq project.metadata[:project_id]
```

Comment:
This new tests is the one that checks for the values coming from Mediaflux (notice the `meta//Title` xpath)